docs/content/examples/tutorials/basic-installation.md

```diff
@@ -84,142 +79,3 @@ services:
       - NET_ADMIN # For Fail2Ban to work
     restart: always
 ```
-
-## A Detailed Example
-
-!!! note
-
-    This is a community contributed guide. Please let us know via a Github Issue if you're having any difficulty following the guide so that we can update it.
-
-This guide is focused on only using [SMTP ports (not POP3 and IMAP)][docs-ports] with the intent to send received mail to another MTA service such as _Gmail_. It is not intended to have a MUA client (_eg: Thunderbird_) to retrieve mail directly from `docker-mailserver` via POP3/IMAP.
-
-In this setup `docker-mailserver` is not intended to receive email externally, so no anti-spam or anti-virus software is needed, making the service lighter to run.
```

Comment:
Kinda worth noting that this is focused mostly on a common scenario of relaying mail for local services in other containers, not a more traditional mail server with inboxes for users to manage mail.

Almost akin to `SMTP_ONLY=1`, but without the issue of unsupported / documented setup. Dovecot can still receive mail bounces I guess and system user mail (_from accounts in DMS like cron_), and provide SASL auth for the external services relaying mail.

Reply:
👍🏼